x/auth/client/cli/tips.go

```diff
@@ -0,0 +1,85 @@
+package cli
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	clienttx "github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/types/tx"
+	authclient "github.com/cosmos/cosmos-sdk/x/auth/client"
+)
+
+func GetAuxToFeeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "aux-to-fee <aux_signed_tx.json>",
+		Short: "includes the aux signer data in the tx, broadcast the tx, and sends the tip amount to the broadcaster",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			auxSignerData := &tx.AuxSignerData{}
+			err = readAuxSignerData(clientCtx.Codec, auxSignerData, args[0])
+			if err != nil {
+				return err
+			}
+
+			f := clienttx.NewFactoryCLI(clientCtx, cmd.Flags())
+
+			txBuilder := clientCtx.TxConfig.NewTxBuilder()
+			err = txBuilder.AddAuxSignerData(*auxSignerData)
```

Comment:
imo it reads better:

```suggestion
			auxSignerData := tx.AuxSignerData{}
			err = readAuxSignerData(clientCtx.Codec, &auxSignerData, args[0])
			if err != nil {
				return err
			}

			f := clienttx.NewFactoryCLI(clientCtx, cmd.Flags())

			txBuilder := clientCtx.TxConfig.NewTxBuilder()
			err = txBuilder.AddAuxSignerData(auxSignerData)
```